conf/modules.config

```
@@ -10,4 +10,14 @@ process {
         ]
     }
 
+
+ withName: MERGE {
+        errorStrategy =  { 'ignore' }
+                memory = { 64.GB }
```

```suggestion
        memory = { 64.GB }
```

Maybe ignoring is not the best solution